### ISSUE TITLE
Keep a copy of Magisk in `addon.d`

### ISSUE THREAD

**Issue by Iey4iej3** [yujincheng08/Magisk-Addon#2]
In order to keep Magisk survive after updates of LOS, since the recovery cannot decrypt `/data` partition, it seems necessary to keep a copy of Magisk in `addon.d`, just as what [Lygisk](https://github.com/programminghoch10/Lygisk/tree/master) does.

I posted https://github.com/programminghoch10/Lygisk/issues/20

**Comment by yujincheng08** [comment]
makes sense. but i hv no idea where to keep the copy of magisk. Any suggestion? I also want to get rid of /data. We can absolutely update the copy of magisk every boot so that we can keep magisk up to date.

**Comment by Iey4iej3** [comment]
I guess that it would be better to wait to see what Lygisk maintainers would comment.
